generators/server/cleanup.js

```diff
@@ -195,6 +191,7 @@ function cleanupOldServerFiles(generator, javaDir, testDir, mainResourceDir, tes
     if (generator.databaseTypeNeo4j) {
       generator.removeFile(`${testDir}AbstractNeo4jIT.java`);
     }
+    generator.removeFile(`${testDir}NoOpMailConfiguration.java`);
   }
   if (generator.isJhipsterVersionLessThan('7.8.2')) {
```

Comment:
```suggestion
  }
  if (generator.isJhipsterVersionLessThan('7.8.2')) {
    generator.removeFile(`${testDir}NoOpMailConfiguration.java`);
```

Reply:
Resolved in https://github.com/Tcharl/generator-jhipster/pull/2, Just waiting ci to pass to merge.
Also took your advice regarding testcontainers and scripts (to keep blueprint using ci scripts).